pkg/kubelet/dockershim/libdocker/fake_client.go

```diff
@@ -735,9 +735,9 @@ func (f *FakeDockerClient) ResetImages() {
 func (f *FakeDockerClient) InjectImageInspects(inspects []dockertypes.ImageInspect) {
 	f.Lock()
 	defer f.Unlock()
-	for _, i := range inspects {
-		f.Images = append(f.Images, *createImageFromImageInspect(i))
-		f.ImageInspects[i.ID] = &i
+	for i := range inspects {
```

Comment:
I'd usually take a single copy of the value here, rather than subscripting in all of the use cases (mostly for ease-of-skimming). Otherwise, the change makes sense to me.

Reply:
Changes done.